consensus/polybft/contracts_initializer.go

```diff
@@ -404,3 +429,8 @@ func callContract(from, to types.Address, input []byte, contractName string, tra
 func isNativeRewardToken(cfg PolyBFTConfig) bool {
 	return cfg.RewardConfig.TokenAddress == contracts.NativeERC20TokenContract
 }
+
+// isNativeStakeToken return true in case a native token is used for staking
+func isNativeStakeToken(cfg PolyBFTConfig) bool {
```

Comment:
Export the function and use it in `polybft_params.go` as well. Also I'd consider changing so that it accepts `stakeTokenAddr` (as little as possible)


```suggestion
func IsNativeStakeToken(stakeTokenAddr types.Address) bool {
```